docs/content/users/quickstart.md

```diff
@@ -47,6 +47,35 @@ ddev launch
     ddev launch
     ```
 
+## Contao
+
+=== "Composer"
```

Comment:
I would remove the tab. contao would be the sole project type that has a single tab. if there is only content for a single tab then place the text as is.

Reply:
Contao offers two install methods:
- Composer
- Contao Manager

I wanted to support both, therefore the tab.

Plus, Contao offers a demo which could be helpful, so there could be three tabs:
- Composer
- Contao Manager
- Demo

Reply:
ahhhh then it is totally fine, i've just noticed the single tab looking at the preview: https://ddev--5672.org.readthedocs.build/en/5672/ but i am not sure if it is necessary to a demo tab which is probably including the link to the demo site? a tab for each of the two install methods (composer and contao manager) is probably enough?

Reply:
I did not know that there is a preview rendered for the docs. Thanks for pointing me to it.

The demo site is something you can install to play around with locally. So, it has different install instructions.